src/index.html

```diff
@@ -21,7 +21,469 @@
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__product-image"
+            src="images/imac.jpeg"
+            alt="product image"
+          />
+
+          <span class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </span>
+
+          <span class="card__code">Product code: 195434</span>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <a
+              class="card__reviews"
+              href="#"
+            >
+              Reviews: 5
+            </a>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+
+            <span class="card__price-amount">$2,199</span>
+          </div>
+
+          <a
+            class="card__button"
+            data-qa="card-hover"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__product-image"
+            src="images/imac.jpeg"
+            alt="product image"
+          />
+
+          <span class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </span>
+
+          <span class="card__code">Product code: 195434</span>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <a
+              class="card__reviews"
+              href="#"
+            >
+              Reviews: 5
+            </a>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+
+            <span class="card__price-amount">$2,199</span>
+          </div>
+
+          <a
+            class="card__button"
+            data-qa="card-hover"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+        <div
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div
```